dev_sandbox/GH_Code/3D_Data_Plot.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# List of unique hexcode colors
+colors = ['#0000FF', '#000000', '#FF0000', '#FFC0CB', '#FFA07A', '#FF7F50',

```

Comment:
comment: Since this is all sandbox code it's not a big deal, but generally it'd be worth it to put this in a `main` method of some sort.